### PR TITLE
oracledb py module

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

Fixes: #1996

##### Component Name

python.d.plugin oracledb module

#### Used queries:

 - [system metrics]:

```sql
SELECT
  metric_name,
  value
FROM
  gv$sysmetric
ORDER BY
  begin_time
```

<details><summary>collected metrics</summary>
<p>

|   name| 
|---|
| average_active_sessions |
| session_count |
| session_limit_percent |
| logons_per_sec |
| physical_reads_per_sec |
| physical_writes_per_sec |
| disk_sort_per_sec |
| long_table_scans_per_sec |
| database_wait_time_ratio |
| shared_pool_free_percent |
| memory_sorts_ratio |
| sql_service_response_time |
| user_rollbacks_per_sec |
| enqueue_timeouts_per_sec |
| buffer_cache_hit_ratio |
| cursor_cache_hit_ratio |
| library_cache_hit_ratio |
| row_cache_hit_ratio |
| global_cache_blocks_corrupted |
| global_cache_blocks_lost |

</p>
</details>
</br>

 - [tablespace]

```sql
SELECT
  m.tablespace_name,
  m.used_space * t.block_size AS used_bytes,
  m.tablespace_size * t.block_size AS max_bytes,
  m.used_percent
FROM
  dba_tablespace_usage_metrics m
  JOIN dba_tablespaces t ON m.tablespace_name = t.tablespace_name
```
<details><summary>collected metrics</summary>
<p>

|   name| 
|---|
| tablespace_size |
| tablespace_used |
| tablespace_used_in_percent |

</p>
</details>
</br>

 - [activities count]

```sql
SELECT
  name,
  value
FROM
  v$sysstat
WHERE
  name IN (
    'parse count (total)',
    'execute count',
    'user commits',
    'user rollbacks'
  )
```
<details><summary>collected metrics</summary>
<p>

|   name| 
|---|
| activity_parse_count |
| activity_execute_count |
| activity_user_commits |
| activity_user_rollbacks |

</p>
</details>
</br>

 - wait time

```sql
SELECT
  n.wait_class,
  round(m.time_waited / m.INTSIZE_CSEC, 3)
FROM
  v$waitclassmetric m,
  v$system_wait_class n
WHERE
  m.wait_class_id = n.wait_class_id
  AND n.wait_class != 'Idle'
```

### CODE REVIEW

**Comment by ilyam8** [comment]
@jurgenhaas @ldod-lle

Hi guys, i need your help to test/improve.

Fastest way to test modue:
 1. install necessary packages
https://github.com/ilyam8/netdata/tree/oracledb/collectors/python.d.plugin/oracledb#prerequisite
2. download module
```
wget https://raw.githubusercontent.com/ilyam8/netdata/oracledb/collectors/python.d.plugin/oracledb/oracledb.chart.py
```
3. move module to `/python.d` (path depends on installation)

```
mv oracledb.chart.py /opt/netdata/usr/libexec/netdata/python.d
```
3. add default your user, password, server and service directly to the code

https://github.com/ilyam8/netdata/blob/c3fc7aab62d11f6571e27d01557e4fefb7b448ca/collectors/python.d.plugin/oracledb/oracledb.chart.py#L284-L287
 
4. restart netdata.service

**Comment by ilyam8** [comment]
@cakrit please approve

**Comment by ilyam8** [comment]
@paulfantom please review/approve too, i can't merge it with your or @ktsaou 	approve

File: collectors/python.d.plugin/oracledb/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+remote:
+  user: 'netdata'
+  password: 'secret'
+  server: '10.0.0.1:1521'
+  service: 'XE'
+```
+
+All parameters are required. Without them module will fail to start.
+
+[![analytics](https://www.google-analytics.com/collect?v=1&aip=1&t=pageview&_s=1&ds=github&dr=https%3A%2F%2Fgithub.com%2Fnetdata%2Fnetdata&dl=https%3A%2F%2Fmy-netdata.io%2Fgithub%2Fcollectors%2Fpython.d.plugin%2Fexample%2FREADME&_u=MAC~&cid=5792dfd7-8dc4-476b-af31-da2fdb9f93d2&tid=UA-64295674-3)]()
```
Comment (paulfantom) [inline collectors/python.d.plugin/oracledb/README.md]: @cakrit won't this cause that page views will be counted towards [python.d.plugin/example](https://docs.netdata.cloud/collectors/python.d.plugin/example/) page?

File: collectors/python.d.plugin/oracledb/oracledb.chart.py [hunk 2]
```diff
@@ -0,0 +1,712 @@
+# -*- coding: utf-8 -*-
+# Description: example netdata python.d module
```
Comment (paulfantom) [inline collectors/python.d.plugin/oracledb/oracledb.chart.py]: not exactly "example" :)

File: collectors/python.d.plugin/oracledb/README.md (hunk 1 above)
Comment (ilyam8) [reply collectors/python.d.plugin/oracledb/README.md]: Opsie, i copied readme from example module

**Comment by ilyam8** [comment]
@cakrit please approve

**Pull request merged**